flux-refineck/src/sigs.rs

```diff
@@ -159,6 +218,59 @@ fn mk_signed_bin_ops() -> impl Iterator<Item = (mir::BinOp, Sig<2>)> {
         })
 }
 
+#[rustfmt::skip]
+fn mk_signed_bin_ops_check_overflow() -> impl Iterator<Item = (mir::BinOp, Sig<2>)> {
+    use mir::BinOp::*;
+    INT_TYS
+        .into_iter()
+        .flat_map(|int_ty| {
+            define_btys! {
+                let bool = BaseTy::Bool;
+                let Int = BaseTy::Int(int_ty);
+            }
+            let bit_width: u128 = int_ty.bit_width().unwrap_or(flux_config::pointer_width().bits()).into();
+            [
+                // ARITH
+                (Add, s!(fn(a: Int, b: Int) -> Int[a + b]
+                            requires E::and(
+                                [E::le(&a + &b, int_max(bit_width)),
+                                E::ge(a + b, int_min(bit_width))]
+                            ) => ConstrReason::Overflow)
```

Comment:
Nit: I would format it like so

```suggestion
                (Add, s!(fn(a: Int, b: Int) -> Int[a + b]
                            requires E::and([
                                         E::le(&a + &b, int_max(bit_width)),
                                         E::ge(a + b, int_min(bit_width)),
                                     ]) => ConstrReason::Overflow)
```